tests/context.rs

```diff
@@ -60,24 +61,28 @@ impl EnvConfig {
             ),
             local_golem: std::env::var("GOLEM_DOCKER_SERVICES").is_err(),
             db_type: DbType::from_env(),
+            n_worker_executors: shards,
         }
     }
+
+    pub fn from_env() -> EnvConfig {
+        Self::from_env_with_shards(3)
+    }
 }
 
 pub struct Context<'docker_client> {
```

Comment:
Always a bit of turn down for me to see connecting to docker from rust.

Reply:
Why?

Reply:
It is for testcontainers